stdlib/2and3/sre_compile.pyi

```diff
@@ -0,0 +1,13 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/sre_compile.py
+# and https://github.com/python/cpython/blob/master/Lib/sre_compile.py
+
+import sys
+from sre_parse import SubPattern
+from typing import Any, Pattern, Tuple, Type, Union
+
+MAXCODE = ...  # type: int
+if sys.version_info < (3, 0):
+    STRING_TYPES = ...  # type: Union[Tuple[Type[str]], Tuple[Type[str], Type[unicode]]]
```

Comment:
I think we don't need to worry about the case where `unicode` does not exist. You can just say `STRING_TYPES = (str, unicode)`, similar to what the stubs for `six` do.